doc/release-notes.md

```diff
@@ -135,6 +135,10 @@ Updated settings
   in future releases. Refer to the help of the affected settings `-whitebind`
   and `-whitelist` for more details. (#19191)
 
+- Netmasks that contain 1-bits after 0-bits (the 1-bits are not contiguous on
+  the left side, e.g. 255.0.255.255) are no longer accepted. They are invalid
+  according to RFC 4632.
```

Comment:
Append " (#19628)"

Reply:
Will do so in a followup, to keep the current ACKs.